site/source/docs/porting/exceptions.rst

```diff
@@ -0,0 +1,50 @@
+.. Exceptions support:
+
+==============================
+C++ exceptions support
+==============================
+
+C++ exceptions support can be enabled in emscripten using ``-fexceptions``.
+Note that unlike on most platforms, C++ exceptions are disabled by default due
+to their current overhead, see later.
+
+Passing ``-fexceptions`` during compilation will pass the relevant flags to
+clang, and during link it also links in system libraries with exceptions
```

Comment:
I'm not a native speaker, but `during link` sounds wrong to me. Would it not be `during linking` or `during link time`?

Reply:
Good question. I'm not a native speaker either, so I'm not sure. Searching, I see some results (200K) that use such a phrase, e.g.

> During link, the object files will all be combined together and the function calls and global variables will be linked. https://electronics.stackexchange.com/questions/108515/purpose-of-arm-linker-files

Seems most common in "shorthand" speech (in a title or summary) like https://gcc.gnu.org/legacy-ml/gcc-bugs/2015-12/msg01508.html and https://www.ibm.com/support/pages/apar/IY50542

I'm not opposed to changing this if it would feel better another way. However I'm unsure myself which of the alternatives would be best? Maybe "during the link stage"?

Reply:
replacing "during compilation` and "during link" with "at compile time" and "at link time" would sound clearest to me fwiw.